src/check_if_venv.py

```diff
@@ -0,0 +1,8 @@
+# from https://stackoverflow.com/questions/1871549/determine-if-python-is-running-inside-virtualenv
+import sys
+
+isVenv = hasattr(sys, "real_prefix") or (
+    hasattr(sys, "base_prefix") and sys.base_prefix != sys.prefix
+)
+
+print(int(isVenv))

```

Comment:
Is this something you want to print?

Reply:
Yes, because the frontend would read the result this way.

Reply:
As of right now, I can't really find another better way of doing this. Is the current way fine?

Reply:
This looks fine to me! I would add a comment to make things more clear, though.

Reply:
ah ok cool! will add

Reply:
added! c:

Reply:
I'd suggest adding a comment explaining that too!